velox/core/SimpleFunctionMetadata.h

```diff
@@ -394,6 +394,7 @@ class ISimpleFunctionMetadata {
   virtual bool physicalSignatureEquals(
       const ISimpleFunctionMetadata& other) const = 0;
   virtual std::string helpMessage(const std::string& name) const = 0;
+  virtual std::string toString(const std::string& name) const = 0;
```

Comment:
Let's rename to toDebugString and remove 'name' parameter.

Let's document this method.